lldb/test/API/lang/cpp/template-arguments/TestCppTemplateArguments.py

```diff
@@ -82,7 +82,7 @@ def test(self):
         value = self.expect_expr("temp7", result_type="Foo<__fp16, __fp16>")
```

Comment:
Shouldn't this one show up as `_Float16` now? Or is there some other place we're missing support for this?